buildscripts/build_deb_pkgs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+export GIT_TAG=$(git describe --tags --long | sed 's/-/\./' | sed 's/-g/-/' | sed 's/-/~/')
+
+if [ "$1" != "--no-compile" ]; then
+    compile_hologram.sh || exit 1
+fi
+
+mkdir -p /hologram-build/{server,agent}/root/usr/local/bin
+mkdir -p /hologram-build/{server,agent}/root/etc/hologram
+mkdir -p /hologram-build/{server,agent}/scripts/
+mkdir -p /hologram-build/{server,agent}/root/etc/init.d/
+
+cp ${HOLOGRAM_DIR}/config/agent.json /hologram-build/agent/root/etc/hologram/agent.json
+cp ${BIN_DIR}/hologram-cli /hologram-build/agent/root/usr/local/bin/
+cp ${BIN_DIR}/hologram-agent /hologram-build/agent/root/usr/local/bin/
+cp ${BIN_DIR}/hologram-authorize /hologram-build/agent/root/usr/local/bin/
+
+cp ${HOLOGRAM_DIR}/agent/support/debian/after-install.sh /hologram-build/agent/scripts/
+cp ${HOLOGRAM_DIR}/agent/support/debian/before-remove.sh /hologram-build/agent/scripts/
+cp ${HOLOGRAM_DIR}/agent/support/debian/init.sh /hologram-build/agent/root/etc/init.d/hologram-agent
+
+cp ${HOLOGRAM_DIR}/config/server.json /hologram-build/server/root/etc/hologram/server.json
+
+cp ${BIN_DIR}/hologram-cli /hologram-build/server/root/usr/local/bin/
+cp ${BIN_DIR}/hologram-server /hologram-build/server/root/usr/local/bin/
+cp ${BIN_DIR}/hologram-authorize /hologram-build/server/root/usr/local/bin/
+
+cp ${HOLOGRAM_DIR}/server/after-install.sh /hologram-build/server/scripts/
+cp ${HOLOGRAM_DIR}/server/before-remove.sh /hologram-build/server/scripts/
+
+cp ${HOLOGRAM_DIR}/server/support/hologram.init.sh /hologram-build/server/root/etc/init.d/hologram-server
```

Comment:
Instead of doing `cp' maybe it's worth using install(1)? With install you can set the permissions of copied files, so you kinda have more control on it, so you could end up with something like: `install -v -m0755 ${BIN_DIR}/hologram-cli /hologram-build/server/root/usr/local/bin/'